spec/features/visitor_can_browse_items_by_brand_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "VisitorCanBrowseItemsByBrands", type: :feature do
+  feature "visitor" do
+    create_category
+    create_brand
+    create_items
+
+    it "can browse items by category" do
+      visit root_path
+      click_link "Canon"
+
+      expect(current_path).to eq(brand_path(brand))
+      within('.items') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.